pkg/minishift/util/set_proxy_to_shell.go

```diff
@@ -1,32 +0,0 @@
-/*
-Copyright (C) 2016 Red Hat, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package util
```

Comment:
I am not convinced that it will useful to anything. May be I am wrong and I am all ears. However if we can not use it we should not keep it in the code right? For historical purpose it is anyway part of the git commit history, so if required we can find it.